dotnet/src/SemanticKernel.Abstractions/Diagnostics/SKException.cs

```diff
@@ -7,20 +7,20 @@ namespace Microsoft.SemanticKernel.Diagnostics;
 /// <summary>
 /// Provides the base exception from which all Semantic Kernel exceptions derive.
 /// </summary>
-public abstract class SKException : Exception
+public class SKException : Exception
 {
     /// <summary>
     /// Initializes a new instance of the <see cref="SKException"/> class with a default message.
     /// </summary>
-    protected SKException()
+    public SKException()
```

Comment:
Update the description? OR What is the default message?

Reply:
Ahh, fixed :)